projects/kit/components/input-inline/input-inline.component.ts

```diff
@@ -70,7 +70,7 @@ export class TuiInputInlineComponent
 
     @HostBinding('attr.data-value')
```

Comment:
@waterplea what do you think?


```ts
@HostBinding('attr.data-value')
    get maskedValue(): string {
        return this.native?.nativeElement.value ?? this.value?.toString();
    }
```

Reply:
I have tried this myself and not working, needs to check the `this.value` which represents the input value.

Reply:
Oh, I see, thank you